site/docs/about/roadmap.mdx

```diff
@@ -53,29 +53,45 @@ For previously released features, see changelogs in the [What is new section](/a
 **Q4 2021**
 
 ~~- Component: Basic tables~~<br/>
-- Component: Breadcrumbs
-- Component: File download
+~~- Icons: Next set of icons~~<br/>
+~~- Update: Koros improvements~~<br/>
+~~- Update: Smaller improvements and fixes to multiple components~~<br/>
 
+
+**Q1 2022**
 - Pattern: Multi-step forms
+- Component: Form stepper
+
+
 - Pattern: Cookie consent
 
```

Comment:
Should we list Cookie consent also as component here?

Reply:
Yes, probably

Reply:
- [ ] add Cookie Consent component in the roadmap